lnrpc/routerrpc/router.proto

```diff
@@ -98,6 +98,15 @@ service Router {
     rpc TrackPayment (TrackPaymentRequest) returns (stream PaymentStatus) {
         option deprecated = true;
     }
+    
+    /**
+    HtlcInterceptor dispatches a bi-directional streaming RPC in which
+    Forwarded HTLC requests are sent to the client and the client responds with
+    a boolean that tells LND if this htlc should be intercepted.
+    In case of interception, the htlc can be either settled, cancelled or
+    resumed later by using the ResolveHoldForward endpoint.
```

Comment:
Hi @roeierez. This description text looks wrong. I don't find any `ResolveHoldForward` endpoint.
AFAICT how to resume the incoming payment is by sending with `action` set to `RESUME` in this gRPC stream.  
Perhaps I'm missing something?

Also by reading the protobuf, it doesn't look lke the client is supposed to respond with a boolean if the HTLC should be intercepted or not (rather it is decided by the `incoming_circuit_key`?)

Reply:
@hsjoberg You are right. Unfortunately this description was left from a previous iteration and is indeed wrong.
The interface was changes and the client should return a ForwardHtlcInterceptResponse with action that maps to one of the follows:

**SETTLE**:  to settle the htlc (a preimage is expected as well).
**FAIL**:  to fail the htlc
**RESUME**: to resume with default behavior.

Reply:
@roeierez Thanks, that makes sense.

Cheers!